manifests/4.6/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -35,29 +42,32 @@ spec:
             description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
             properties:
               inputs:
-                description: Inputs are named inputs of log messages
+                description: "Inputs are named filters for log messages to be forwarded.
+                  \n There are three built-in inputs named `application`, `infrastructure`
+                  and `audit`. You don't need to define inputs here if those are sufficient
+                  for your needs. See `inputRefs` for more."
                 items:
                   description: InputSpec defines a selector of log messages.
                   properties:
                     application:
-                      description: 'Enable `application` logs. Use `application: {}`
-                        to enable with no filter.'
+                      description: 'Application enables `application` logs. `application:
```

Comment:
the first part is a little confusing to read simply because the property name is self documenting, what about something like "This property enables...."

Reply:
Reworded thusly: 
	// Application if present enables `application` logs.

This also complies with the Go convention of starting a doc comment with the field name. 
It seems like most Go-based CRDs follow the Go comment convention even though it uses the wrong case (leading caps) for the generated CRD field name. I'll follow the herd on this one, I guess people are used to seeing CRD descriptions with capitalized field names.